src/Concerns/SerializesConnections.php

```diff
@@ -15,7 +15,6 @@ public function __serialize(): array
     {
         return [
             'id' => $this->id(),
-            'identifier' => $this->identifier(),
```

Comment:
Why was this removed?

Reply:
Please mark as ready for review when updated.

Reply:
`identifier` isn't a direct property of the `Connection` object being serialized, but was incorrectly being set as such when the object gets unserialized. This was causing PHPStan to fail static analysis.

The socket's raw `identifier` is resolved from the underlying `WebSocketConnection` which is passed in `Connection`'s constructor. For this reason, it was removed from the `Connection` serialization, which currently only stores the "normalized" socket ID as one of its properties.

Maybe this PR needs more work to try and resurrect the entire connection when unserialized. Although that could be quite a rabbit hole.